Improve wayfinding for various Volto audiences (#5809)
plone/volto: docs/source/index.md

Volto is a React-based frontend for the [Plone CMS](https://plone.org).
It is the default frontend starting with the Plone 6 release.

Volto provides an attractive proposition: integration with the modern frontend development world, access to the huge ecosystem of React libraries and add-ons, combined with the ability to use the mature Plone CMS backend as a development platform.

Thanks to the use of the Plone REST API, it is fully compatible with Plone's content type framework.
But its power comes from the innovative Pastanaga editor, which provides a flexible WYSIWYG editing interface based on "blocks".

Thanks to their simplicity and easy access to advanced frontend integration, Volto blocks can provide a fast development experience that reduces developer frustration and improves end-user experience.

## Get started

Choose from the following sections to begin your journey with Volto.

### Integrators

> An integrator is someone who uses Volto in their client projects.

Reviewer: how about
```suggestion
An integrator is someone who uses Volto to build a website.
```

It can be for yourself, rather than for a client...

Author: As well as for an employer. However, which term would better describe Plone?

- website
- web application

I lean toward the latter, as it implies more than just static content, but includes workflows and moving data from point A to B.

Reviewer: @stevepiercy Not sure we want to get into that here. Just say "to build a project"?

Author: Yup, that works.

Author: Fixed in [ce5da24](https://github.com/plone/volto/pull/5809/commits/ce5da243151845ffd060e11e42798c18c734f456)
